sig-security-tooling/cve-feed/README.md

```diff
@@ -1,48 +1,119 @@
 # Official CVE Feed
 
-The official CVE feed is separated into two main components:
-1. The scripts, that update a cloud bucket containing the feed.
-2. The website, rendering and serving the feed in various formats.
+## Introduction
 
-## Scripts
+The **Kubernetes Official CVE Feed** provides an authoritative and
+machine-readable source of information about security vulnerabilities
+(CVEs) affecting Kubernetes. It helps developers, operators, and
+security professionals stay informed about officially recognized and
+triaged vulnerabilities in the Kubernetes ecosystem.
 
-A script in the [kubernetes/sig-security](https://github.com/kubernetes/sig-security)
-repository under the [sig-security-tooling/cve-feed/hack](https://github.com/kubernetes/sig-security/tree/main/sig-security-tooling/cve-feed/hack)
-folder. This script is
-a bash script named `fetch-cve-feed.sh` that:
-- sets up the python3 environment;
+This feed is maintained by the **Kubernetes Security Response Committee
+(SRC)** and **SIG Security**, ensuring that all CVE data published is
+verified and accurately reflects the current security state of
+Kubernetes.
+
+### Who Uses the Official CVE Feed
+
+The feed serves multiple audiences:
+
+- **Developers and maintainers** — to identify and track vulnerabilities
+  related to their components or dependencies.
+- **Security teams and researchers** — to integrate verified Kubernetes
+  CVE data into vulnerability management tools or monitoring systems.
+- **Cloud providers and Kubernetes distributors** — to automate tracking
+  of upstream advisories and coordinate patching.
+- **End-users and operators** — to monitor official Kubernetes CVEs and
+  apply security updates or mitigations accordingly.
+
+### What It Provides
+
+The Official Kubernetes CVE Feed is distributed in multiple formats:
+
+- **HTML** — a human-readable web page available at  
+  [k8s.io/docs/reference/issues-security/official-cve-feed](https://kubernetes.io/docs/reference/issues-security/official-cve-feed/)
+- **JSON** — the original, machine-readable feed listing all issues
+  labeled as
+  [`official-cve-feed`](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+label%3Aofficial-cve-feed+)
+  in the
+  [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)
+  repository
+- **RSS** — a syndication-friendly view for integration into monitoring
+  or alerting tools
+
+Each entry in these feeds corresponds to an official CVE affecting
+Kubernetes.
+
+## The official CVE feed is separated into two main components
+
+1. The scripts that update a cloud bucket containing the feed.
+2. The website rendering and serving the feed in various formats.
+
+
+### Scripts
+
+A script in the
+[kubernetes/sig-security](https://github.com/kubernetes/sig-security)
```

Comment:
it's okay to go over 80 chars when you have links if you want btw, not a stricti rule, the wrapping is just to ease reviewing and git history change by line
```suggestion
A script in the [kubernetes/sig-security](https://github.com/kubernetes/sig-security)
```